controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -630,15 +630,15 @@ bool master_thread::autoconfig_wsc_add_m2(std::shared_ptr<ieee1905_1::tlvWscM1>
     // Create ConfigData
     uint8_t buf[1024];
     WSC::cConfigData config_data(buf, sizeof(buf), false, true);
-    if (bss_info_conf && !bss_info_conf->ssid.empty() && !bss_info_conf->network_key.empty()) {
+    if (bss_info_conf) {
         config_data.set_ssid(bss_info_conf->ssid);
         config_data.authentication_type_attr().data = bss_info_conf->authentication_type;
         config_data.encryption_type_attr().data     = bss_info_conf->encryption_type;
         config_data.set_network_key(bss_info_conf->network_key);
         config_data.multiap_attr().subelement_value = bss_info_conf->bss_type;
 
         LOG(DEBUG) << "WSC config_data:" << std::hex << std::endl
-                   << "     ssid: " << config_data.ssid() << std::endl
+                   << "     ssid: " << config_data.ssid_str() << std::endl
```

Comment:
I would have done this change in a separate commit, but again that's nitpicking.